index.html

```diff
@@ -1,239 +1,313 @@
-              <i class="material-icons">arrow_forward</i>
-            </button>
-          </div>
-        </section>
-        <section class="card">
-          <div class="img-three img"></div>
-          <div class="project">
-            <p>Project name goes here</p>
-            <ul class="language">
-              <li>HTML/CSS</li>
-              <li>Ruby on Rails</li>
-              <li>JavaScript</li>
-            </ul>
-            <button class="btn">
-              <p>See this project</p>
-              <i class="material-icons">arrow_forward</i>
-            </button>
-          </div>
-        </section>
-        <section class="card">
-          <div class="img-four img"></div>
-          <div class="project">
-            <p>Project name goes here</p>
-            <ul class="language">
-              <li>HTML/CSS</li>
-              <li>Ruby on Rails</li>
-              <li>JavaScript</li>
-            </ul>
-            <button class="btn">
-              <p>See this project</p>
-              <i class="material-icons">arrow_forward</i>
-            </button>
-          </div>
-        </section>
-        <section class="card">
-          <div class="img-five img"></div>
-          <div class="project">
-            <p>Project name goes here</p>
-            <ul class="language">
-              <li>HTML/CSS</li>
-              <li>Ruby on Rails</li>
-              <li>JavaScript</li>
-            </ul>
-            <button class="btn">
-              <p>See this project</p>
-              <i class="material-icons">arrow_forward</i>
-            </button>
-          </div>
-        </section>
-        <section class="card">
-          <div class="img-six img"></div>
-          <div class="project">
-            <p>Project name goes here</p>
-            <ul class="language">
-              <li>HTML/CSS</li>
-              <li>Ruby on Rails</li>
-              <li>JavaScript</li>
-            </ul>
-            <button class="btn">
-              <p>See this project</p>
-              <i class="material-icons">arrow_forward</i>
-            </button>
-          </div>
-          </div>
-        </section>
-      </section>
-      <section class="about-me">
-      <div class="about">
-          <h2>About</h2>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incidiunt ut labore et dolore magna aliqua. Ut enim
-            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-            aliquip ea eiusmod
-          </p>
-          <button type="button" class="about-btn">
-            Get my CV
-            <i class="material-icons">login</i>
-          </button>
+      <!-- <div class="modal">
+        <p>Project name goes here</p>
+        <button>html/css</button>
+        <button>Ruby on rails</button>
+        <button>Javascript</button>
+        <div>
+          <img src="./images/Group 109.svg" alt="modalimage">
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is regarded as a bad practice.